proto/ffi.proto

```diff
@@ -61,3 +61,9 @@ message ValidationResult {
   int64 socket_write = 9;  // cbpf only
   int64 socket_read = 10;  // cbpf only
 }
+
+message EncodedProgram {
```

Comment:
Maybe add a comment detailing each of the members?

Reply:
Done